gramex/ml_api.py

```diff
@@ -426,3 +435,49 @@ def _predict(
     ):
         text = X["text"]
         return self.model.predict(text)
+
+
+class KerasApplications(AbstractModel):
+    def __init__(self, model, params=None, data=None, **kwargs):
+        self.model = model
+        if params is None:
+            params = {}
+        self.params = params
+        self.kwargs = kwargs
+
+    @classmethod
+    def from_disk(cls, path, klass):
+        # Load model from disk
+        return cls
+
+    def predict(self, data=None, **kwargs):
+        from tensorflow.keras.preprocessing import image
+
+        mclass, wrapper = search_modelclass(kwargs['mclass'])
+        module_imp = __import__(mclass.__module__, fromlist=SEARCH_MODULES[wrapper])
```

Comment:
None of this is required. The `get_model` does all of this.

Reply:
Okay I guess what you are trying to do is as follows. If we have some Keras app, then the `preprocess_input` and `decode_predictions` should come from the same module.  This can be done as follows:
```python
from gramex.config import locate
preprocess_input = locate('preprocess_input', [model.__module__])
decode_predictions = locate('decode_predictions', [model.__module__])
```

(where model is the actual Keras model)

Reply:
The point is to not import the whole module and only get the functions we need.